common/src/main/java/org/apache/atlas/repository/Constants.java

```diff
@@ -75,6 +75,7 @@ public final class Constants {
     public static final String TYPEVERSION_PROPERTY_KEY     = getEncodedTypePropertyKey(INTERNAL_PROPERTY_KEY_PREFIX + "type.version");
     public static final String TYPEOPTIONS_PROPERTY_KEY     = getEncodedTypePropertyKey(INTERNAL_PROPERTY_KEY_PREFIX + "type.options");
     public static final String TYPESERVICETYPE_PROPERTY_KEY = getEncodedTypePropertyKey(INTERNAL_PROPERTY_KEY_PREFIX + "type.servicetype");
+    public static final String TYPE_SKIP_DISPLAYNAME_UNIQUENESS_CHECK = getEncodedTypePropertyKey(INTERNAL_PROPERTY_KEY_PREFIX + "type.skipDisplayNameUniquenessCheck");
```

Comment:
I could not see where we actually skip checking duplicate display name based on this flag

Reply:
In `AtlasClassificationDefStoreV2.preCreate` I've added a condition if `skipDisplayNameUniquenessCheck = true` then it'll skip checking display name.
Following has been released on prod
https://github.com/atlanhq/atlas-metastore/pull/1473/files

But found a bug that `skipDisplayNameUniquenessCheck` is not stored in graph store. So made changes to fix this bug.

@nikhilbonte21